sql/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -40,7 +39,6 @@
 import static org.hamcrest.Matchers.*;
 import static org.hamcrest.core.Is.is;
 
-@ESIntegTestCase.ClusterScope(numDataNodes = 2, numClientNodes = 0, randomDynamicTemplates = false)
```

Comment:
Does it make sense to keep annotation and set minNumDataNodes=2 and have a separate class with numDataNodes=1 like we do with `JoinIntegrationTest` & `JoinSingleNodeIntegrationTest`